app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str):
+    command_list = command.split(" ")
+    if command_list[0] != "cp":
+        print("Enter correct command (cp)")
+    elif command_list[1] != command_list[2]:
+        with open(command_list[1], "r") as file:
```

Comment:
Use more descriptive names for the files variables

Reply:
Thx) I fix it all.